packages/fractal/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@frctl/fractal",
+  "version": "1.3.0",
+  "description": "A tool to help you build and document web component libraries and then integrate them into your projects.",
+  "main": "src/fractal.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/frctl/fractal.git",
+    "directory": "packages/fractal"
+  },
+  "author": "Mark Perkins <mark@clearleft.com> (http://clearleft.com/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/frctl/fractal/issues"
+  },
+  "homepage": "https://github.com/frctl/fractal",
+  "dependencies": {
+    "@allmarkedup/fang": "^1.0.0",
+    "@frctl/handlebars": "^1.2.0",
+    "@frctl/mandelbrot": "^1.3.0",
```

Comment:
Should we update to 1.4.0 ?

Reply:
I updated the version in the mandelbrot import PR